### ISSUE TITLE
Targets as gauge

### ISSUE THREAD

**Issue by nkm8** [rsnodgrass/hass-poolmath#6]
Great job on the add-on -very useful!

I am new to Home Assistant development, but I wanted to see if there is anything I can do to help. Specifically, I noticed you added different targets/ranges: https://github.com/rsnodgrass/hass-poolmath/blob/master/custom_components/poolmath/targets.yaml

Are these values available in HA yet? I could see them as great way to configure different UI elements, like Gauges, with red/yellow/green configured based on a selected target for each sensor: https://www.home-assistant.io/lovelace/gauge/

**Comment by rsnodgrass** [comment]
Yes, my thought was to allow the user to select a target (defaulting to Trouble Free Pool's Pool math targets) and add them as attributes of each sensor.  This should allow using them in the Lovelace UI in various visualizations of gauges, bar charts, etc.  However, some Lovelace cards may not grab data from attributes for the visualizations so may need to create separate 'input_numbers' (rather than sensors I think) for the target values in the short term.

Feel free to contribute!  I'm not planning on adding this anytime soon.

**Comment by rsnodgrass** [comment]
I did split out the target profiles into separate yaml files under target/.

**Comment by rsnodgrass** [comment]
I flushed out some structure for targets, and a hardcoded 'tfp' targets value.  These should appear in attributes now for each sensor.  Not sure if this is useful, feel free to play around and see if this can be used by gauges.  I think this may be too simplistic, and some targets really are calculations (based on other attributes such as cya) and not fixed values.

Look forward to any contributions to targets and if they are useful.  I'm done playing with this for now.
